src/utils/fd-axios.js

```diff
@@ -7,7 +7,7 @@ let FD_API_URL
 if (window.location.host === 'fd-vue-webapp.herokuapp.com') {
   FD_API_URL = 'http://174.138.4.3:9111/v3'
 } else {
-  FD_API_URL = 'http://' + window.location.hostname + ':9111/v3'
+  FD_API_URL = 'http://174.138.4.3:9111/v3'
```

Comment:
Is this change here actually relevant? I think this shall introduce a wrong behavior.

Reply:
Fixed in 2cb69782bd6cb703fe3a44e0d250aac56db8fa48

Reply:
:+1: